desktop/src/player.rs

```diff
@@ -184,7 +185,9 @@ impl ActivePlayer {
             .unwrap_or_else(|| movie_url.as_str())
             .to_string();
 
-        window.set_title(&format!("Ruffle - {name}"));
+        let readable_name = decode(&name).expect("UTF-8");
```

Comment:
Hmm, this probably shouldn't panic (a SWF file with an invalid name should still play). Can this be something like `unwrap_or(name)`?